app/models/manageiq/providers/openstack/infra_manager/host.rb

```diff
@@ -412,11 +412,119 @@ def provide(taskid = nil)
       begin
         MiqEvent.raise_evm_job_event(self, :type => "provide", :suffix => "complete")
       rescue => err
-        $log.warn("Error raising complete provide event for #{log_target}: #{err.message}")
+        _log.warn("Error raising complete provide event for #{log_target}: #{err.message}")
       end
     end
 
     task.update_status("Finished", task_status, "Provide Complete with #{workflow_state}") if task
-    $log.info("Provide #{log_target}...Complete - Timings: #{t.inspect}")
+    _log.info("Provide #{log_target}...Complete - Timings: #{t.inspect}")
+  end
+
+  def supports_start?
+    (state.casecmp("off") == 0 && hardware.provision_state == "active")
+  end
+
+  def validate_start
+    if supports_start?
+      {:available => true,   :message => nil}
+    else
+      {:available => false,  :message => _("Cannot start. Already on.")}
```

Comment:
Good point. I removed the checks on provision_state. So it should be correct now.